cmd/manager/operator.go

```diff
@@ -179,6 +179,9 @@ func RunOperator(cmd *cobra.Command, args []string) {
 	} else {
 		setupLog.Info("Watching all namespaces")
 	}
+
+	// Unused, but kept until we decide what to do with multi-namespace support. See the creation of `mgr` below for
+	// the actual operator options configuration.
```

Comment:
Not strictly related to this patch (which looks good to me btw) but could we even do anything else than pass the namespace (and optionally the `NewCache` param) to `ctrl.NewManager` ?